### PR TITLE
Enhanced Oracle automation

### PR DESCRIPTION
**Pull request by shubham-yb**
- Improved the validations for some tests
- Added sequences automation
- Added constraints automation
- Added views automation
- Added Case Sensitive Tables/Columns
- Removed unnecessary code 
- Updated docker images

### CODE REVIEW

File: migtests/tests/mysql/reserved-words/ReservedWords.sql [hunk 1]
```diff
@@ -52,4 +54,39 @@ INSERT into `check` values(1, 'abc');
 INSERT into `check` values(2, 'abc');
 INSERT into `check` values(3, 'abc');
 INSERT into `check` values(4, 'abc');
-INSERT into `check` values(5, 'abc');
\ No newline at end of file
+INSERT into `check` values(5, 'abc');
+
+drop table if exists `Mixed_Case_Test`;
+
+CREATE TABLE `Mixed_Case_Test` (
```
Comment (makalaaneesh) [inline migtests/tests/mysql/reserved-words/ReservedWords.sql:61]: Assuming this is mixed case just for the table name and not the columns, can you rename it to `Mixed_Case_Table_Name_Test` or something like that just to make it more explicit?

File: migtests/tests/oracle/datatypes/validate [hunk 2]
```diff
@@ -22,6 +22,30 @@ EXPECTED_ROW_COUNT = {
...
+	},
+	'number_ps': {
+		'num_val': '123.89',
+		'num_3': '402',
+		'num_p_s': '5.79',
+		'num_s_p': '0.0000012',
+		'num_neg_s': '546.99'
+	}
+}
+if os.environ.get('BETA_FAST_DATA_EXPORT') == '1':
```
Comment (makalaaneesh) [inline migtests/tests/oracle/datatypes/validate:40]: Why is this different for debezium?

File: migtests/tests/oracle/datatypes/validate
Comment (makalaaneesh) [inline migtests/tests/oracle/datatypes/validate:1]: Can you add a TODO in this file/create a jira to track the task of more rigorous datatype checks for oracle?

File: migtests/tests/oracle/misc-tests/misc_tests_schema_data.sql [hunk 3]
```diff
@@ -95,4 +95,34 @@ INSERT into "check" values(1, 'abc');
 INSERT into "check" values(2, 'abc');
 INSERT into "check" values(3, 'abc');
 INSERT into "check" values(4, 'abc');
-INSERT into "check" values(5, 'abc');
\ No newline at end of file
+INSERT into "check" values(5, 'abc');
+
+CREATE TABLE "Mixed_Case_Test" (
```
Comment (makalaaneesh) [inline migtests/tests/oracle/misc-tests/misc_tests_schema_data.sql:100]: same comment as for mysql

**Review by makalaaneesh — COMMENTED** [review]
Gave it a high-level look. As long as all tests are passing, this should be good! (pls share the jenkins pipeline)
Had one question about the debezium specific validation.

File: migtests/tests/oracle/datatypes/validate (hunk 2 above)
Comment (makalaaneesh) [reply migtests/tests/oracle/datatypes/validate:40]: EXPECTED_SUM_OF_COLUMN[numeric_types][bin_float_val]

Comment (makalaaneesh) [reply migtests/tests/oracle/datatypes/validate:40]: Can you pls add a comment explaining why we're doing this, and why only this column? We'll forget later :D

Comment (makalaaneesh) [reply migtests/tests/oracle/datatypes/validate:40]: Also, we need to add asserts for table `number_ps` as well. 
(It's okay if you want to do it in a separate PR, in which case add a TODO comment here)

**Review by makalaaneesh — APPROVED** [review]
LGTM with couple of comments

**Comment by shubham-yb** [comment]
Jenkins builds:
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline-test/128/
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline-test/127/

**Pull request merged**